src/main/resources/application.properties

```diff
@@ -35,13 +35,20 @@ mybatis.type-aliases-package=io.streamnative.pulsar.manager
 #spring.datasource.password=
 
 # HerdDB JDBC Driver
- spring.datasource.driver-class-name=herddb.jdbc.Driver
- HerdDB - local in memory-only (check docs for starting an embedded server or embedded replicated server)
- spring.datasource.url=jdbc:herddb:server:localhost:7000?server.base.dir=.
- spring.datasource.schema=classpath:/META-INF/sql/herddb-schema.sql
- spring.datasource.username=sa
- spring.datasource.password=hdb
- spring.datasource.initialization-mode=always
+spring.datasource.driver-class-name=herddb.jdbc.Driver
+# HerdDB - local in memory-only
+#spring.datasource.url=jdbc:herddb:local
+# HerdDB - start embedded server, data persisted on local disk (directory 'dbdata'), listening on localhost:7000
+spring.datasource.url=jdbc:herddb:server:localhost:7000?server.start=true&server.base.dir=dbdata
```

Comment:
this is the new default configuration:
start embedded server, data persisted on local disk (directory 'dbdata'), listening on localhost:7000

this way the system persist data across restart, like it was with SQLLite, in my previous commit the default configuration needed a running HerdDB standalone server running locally